clash-prelude/src/Clash/Sized/Internal/BitVector.hs

```diff
@@ -553,6 +557,68 @@ read# cs0 = (fromIntegral (length cs1), BV m v)
            "Clash.Sized.Internal.bLit: unknown character: "
         ++ show c ++ " in input: " ++ cs0
 
+-- | Create a hexadecimal literal.
+-- ghci> $(hLit "dead")
+-- 0b1101_1110_1010_1101
+-- it :: BitVector 16
+--
+-- Don't care digits set 4 bits in the undefined mask.
```

Comment:
```suggestion
-- | Create a hexadecimal literal
--
-- >>> $(hLit "dead")
-- 0b1101_1110_1010_1101
--
-- Don't care digits set 4 bits:
--
-- >>> $(hLit "de..")
-- 0b1101_1110_...._....
```

Reply:
Writing it this way makes it get picked up by [doctest](https://github.com/martijnbastiaan/doctest-parallel/)